README.md

```diff
@@ -18,6 +18,41 @@ canonical sqlite store.
 For making use of a Lair keystore in a client application, see the
 [lair_keystore_api](https://crates.io/crates/lair_keystore_api) crate.
 
+## What is lair-keystore, and why does it exist?
+
+Lair Keystore is a general asymmetric cryptographic private key store
+project originally written for Holochain, but intended to be usable for
+any application.
+
+The store mainly tracks the "seed" data that for ed25519 and x25519 allow
+generation of keypairs, and can be thought of as synonymous with private
+keys.
+
+Lair allows derivation of this seed material for usage similar to HD
+wallets, with the intention that an end-user could create a "root" seed,
+from which could be deterministically derived a revocation seed and any
+number of device and application seeds, which would all be retrievable from
+a securely stored paper mnemonic of the root. (This has not yet been
+implemented in Holochain).
```

Comment:
Are there examples of doing this in Lair? Either as examples or tests? 

I'm not always convinced when I look at this that I understand the how, even though I think I understand the theory.

Reply:
That's a good idea, I'll put in an example of how this might work.

Reply:
Done: https://github.com/holochain/lair/pull/147/commits/04e566bac463344fd8e13fd2c7c32308ece2e890